ats/pkb/query_evaluator.py

```diff
@@ -119,6 +119,8 @@ def resolve_node(param):
         if isinstance(param, int):
             return context["statements"][param]
         else:
+            if param == relation["parameters"][0]:
+                return context["procedures"][param[1:-1]]
```

Comment:
To tez moze byc stmt....

https://github.com/wvffle/ATS-project-simple/blob/master/ats/pql/pql.py#L25-26